plugins/modules/database/misc/redis_info.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Pavlo Bashynskyi (@levonet)
```

Comment:
```suggestion
# Copyright: (c) 2020, Pavlo Bashynskyi (@levonet) <your@mail.com>
```
Maybe one day you'll decide to redirect all ansible stuff to trash:) which sometimes generated automatically in huge number. So, users can find you by email directly. Not necessary, just a suggestion.

we almost finished:) i think:)
@felixfontein , do you have any questions/suggestions?
becides the above i don't

Reply:
I'm not hiding)
But everything is possible in this life.
Thanks for the review, now I know how to run integration tests.